lib/model/query/submissions.js

```diff
@@ -386,14 +398,87 @@ const getForExport = (formId, instanceId, draft, options = QueryOptions.none) =>
   maybeOne(_export(formId, draft, [], options.withCondition({ 'submissions.instanceId': instanceId })))
     .then(map(_exportUnjoiner));
 
+////////////////////////////////////////////////////////////////////////////////
+// DELETE SUBMISSION
+
+const del = (submission) => ({ run }) =>
+  run(markDeleted(submission));
+
+del.audit = (submission, form) => (log) => log('submission.delete', submission.with({ acteeId: form.acteeId }), { instanceId: submission.instanceId });
```

Comment:
`submission.with({ acteeId: form.acteeId }),` can probably just be `{ acteeId: form.acteeId }`

remove `submission.delete` and `submission.restore` from non-verbose filter and include in submission filter